### ISSUE TITLE
Update from 0.28.4 to 0.28.5 fail: Query expansion failed: could not find table

### ISSUE THREAD

**Issue by zaiddabaeen** [metabase/metabase#7303]
I have just attempted to update my 0.27.2 Metabase to 0.28.5 on my EC2 Metabase instance. After running the service, it shut down while the loading bar was showing on the webpage "Embiggenning Data". On running `java -jar metabase.jar`, the log shows:

```
user@ip-172-31-4-190:~$ java -jar metabase.jar 
04-09 09:17:33 INFO metabase.util :: Loading Metabase...
04-09 09:17:38 INFO util.encryption :: DB details encryption is DISABLED for this Metabase instance. 🔓 
See http://www.metabase.com/docs/latest/operations-guide/start.html#encrypting-your-database-connection-details-at-rest for more information.
04-09 09:17:44 INFO metabase.core :: Starting Metabase in STANDALONE mode
*****************************************************************************************************************************************************
*
* DEPRECATION WARNING: You are currently running JDK '1.7.0_131'. Support for Java 7 has been deprecated and will be dropped from a future release.
* See the operation guide for more information: https://metabase.com/docs/latest/operations-guide/start.html.
*
*****************************************************************************************************************************************************
04-09 09:17:44 WARN metabase.core :: [DEPRECATION WARNING: You are currently running JDK '1.7.0_131'. Support for Java 7 has been deprecated and will be dropped from a future release. See the operation guide for more information: https://metabase.com/docs/latest/operations-guide/start.html.]
04-09 09:17:44 INFO metabase.core :: Launching Embedded Jetty Webserver with config:
 {:port 3000}

04-09 09:17:44 INFO metabase.core :: Starting Metabase version v0.28.5 (fa5cb8a release-0.28.5) ...
04-09 09:17:44 INFO metabase.core :: System timezone is 'Etc/UTC' ...
04-09 09:17:46 INFO metabase.core :: Setting up and migrating Metabase DB. Please sit tight, this may take a minute...
04-09 09:17:46 INFO metabase.db :: Verifying mysql Database Connection ...
04-09 09:17:46 INFO metabase.db :: Verify Database Connection ...  ✅
04-09 09:17:46 INFO metabase.db :: Running Database Migrations...
04-09 09:17:46 INFO metabase.db :: Setting up Liquibase...
04-09 09:17:46 INFO metabase.db :: Liquibase is ready.
04-09 09:17:46 INFO metabase.db :: Checking if Database has unrun migrations...
04-09 09:17:50 INFO metabase.db :: Database Migrations Current ...  ✅
com.mchange.v2.cfg.DelayedLogItem [ level -> FINE, text -> "The configuration file for resource identifier 'hocon:/reference,/application,/c3p0,/' could not be found. Skipping.", exception -> null]
04-09 09:17:50 INFO db.migrations :: Running all necessary data migrations, this may take a minute.
04-09 09:17:50 INFO db.migrations :: Running data migration 'repopulate-card-read-permissions'...
04-09 09:17:51 WARN middleware.expand :: The syntax for datetime-field has changed in MBQL '98. [:datetime-field <field> :as <unit>] is deprecated. Prefer [:datetime-field <field> <unit>] instead.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (555) is deprecated in MBQL '98. Please use [:field-id 555] instead.
04-09 09:17:51 WARN middleware.expand :: The syntax for datetime-field has changed in MBQL '98. [:datetime-field <field> :as <unit>] is deprecated. Prefer [:datetime-field <field> <unit>] instead.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (556) is deprecated in MBQL '98. Please use [:field-id 556] instead.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (396) is deprecated in MBQL '98. Please use [:field-id 396] instead.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (427) is deprecated in MBQL '98. Please use [:field-id 427] instead.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (427) is deprecated in MBQL '98. Please use [:field-id 427] instead.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (399) is deprecated in MBQL '98. Please use [:field-id 399] instead.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (399) is deprecated in MBQL '98. Please use [:field-id 399] instead.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (169) is deprecated in MBQL '98. Please use [:field-id 169] instead.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (163) is deprecated in MBQL '98. Please use [:field-id 163] instead.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (163) is deprecated in MBQL '98. Please use [:field-id 163] instead.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (171) is deprecated in MBQL '98. Please use [:field-id 171] instead.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (171) is deprecated in MBQL '98. Please use [:field-id 171] instead.
04-09 09:17:51 WARN middleware.expand :: Specifying :rows as the aggregation type is deprecated in MBQL '98. This is the default behavior, so you don't need to specify it.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (169) is deprecated in MBQL '98. Please use [:field-id 169] instead.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (161) is deprecated in MBQL '98. Please use [:field-id 161] instead.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (163) is deprecated in MBQL '98. Please use [:field-id 163] instead.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (163) is deprecated in MBQL '98. Please use [:field-id 163] instead.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (171) is deprecated in MBQL '98. Please use [:field-id 171] instead.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (171) is deprecated in MBQL '98. Please use [:field-id 171] instead.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (399) is deprecated in MBQL '98. Please use [:field-id 399] instead.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (399) is deprecated in MBQL '98. Please use [:field-id 399] instead.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (427) is deprecated in MBQL '98. Please use [:field-id 427] instead.
04-09 09:17:51 WARN middleware.expand :: Referring to fields by their bare ID (427) is deprecated in MBQL '98. Please use [:field-id 427] instead.
java.lang.Exception: Query expansion failed: could not find table 26 (FK ID = 418). Resolved tables ([fk-id table-id]): ([nil 47])
	at metabase.query_processor.middleware.resolve$field_resolve_table.invokeStatic(resolve.clj:148)
	at metabase.query_processor.middleware.resolve$field_resolve_table.invoke(resolve.clj:131)
	at metabase.query_processor.middleware.resolve$fn__27905$G__27876__27912.invoke(resolve.clj:83)
	at metabase.query_processor.middleware.resolve$resolve_tables$fn__28060.invoke(resolve.clj:403)
	at clojure.walk$walk.invokeStatic(walk.clj:48)
	at clojure.walk$postwalk.invokeStatic(walk.clj:52)
	at clojure.walk$postwalk.invoke(walk.clj:52)
	at clojure.core$partial$fn__4759.invoke(core.clj:2515)
	at clojure.core$map$fn__4785.invoke(core.clj:2644)
	at clojure.lang.LazySeq.sval(LazySeq.java:40)
	at clojure.lang.LazySeq.seq(LazySeq.java:49)
	at clojure.lang.RT.seq(RT.java:521)
	at clojure.core$seq__4357.invokeStatic(core.clj:137)
	at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:24)
	at clojure.core.protocols$fn__6738.invokeStatic(protocols.clj:75)
	at clojure.core.protocols$fn__6738.invoke(protocols.clj:75)
	at clojure.core.protocols$fn__6684$G__6679__6697.invoke(protocols.clj:13)
	at clojure.core$reduce.invokeStatic(core.clj:6545)
	at clojure.core$into.invokeStatic(core.clj:6610)
	at clojure.walk$walk.invokeStatic(walk.clj:49)
	at clojure.walk$postwalk.invokeStatic(walk.clj:52)
	at clojure.walk$postwalk.invoke(walk.clj:52)
	at clojure.core$partial$fn__4759.invoke(core.clj:2515)
	at clojure.core$map$fn__4785.invoke(core.clj:2644)
	at clojure.lang.LazySeq.sval(LazySeq.java:40)
	at clojure.lang.LazySeq.seq(LazySeq.java:49)
	at clojure.lang.RT.seq(RT.java:521)
	at clojure.lang.LazilyPersistentVector.create(LazilyPersistentVector.java:44)
	at clojure.core$vec.invokeStatic(core.clj:377)
	at clojure.walk$walk.invokeStatic(walk.clj:45)
	at clojure.walk$postwalk.invokeStatic(walk.clj:52)
	at clojure.walk$postwalk.invoke(walk.clj:52)
	at clojure.core$partial$fn__4759.invoke(core.clj:2515)
	at clojure.core$map$fn__4785.invoke(core.clj:2646)
	at clojure.lang.LazySeq.sval(LazySeq.java:40)
	at clojure.lang.LazySeq.seq(LazySeq.java:49)
	at clojure.lang.Cons.next(Cons.java:39)
	at clojure.lang.RT.next(RT.java:688)
	at clojure.core$next__4341.invokeStatic(core.clj:64)
	at clojure.core.protocols$fn__6755.invokeStatic(protocols.clj:168)
	at clojure.core.protocols$fn__6755.invoke(protocols.clj:124)
	at clojure.core.protocols$fn__6710$G__6705__6719.invoke(protocols.clj:19)
	at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:31)
	at clojure.core.protocols$fn__6738.invokeStatic(protocols.clj:75)
	at clojure.core.protocols$fn__6738.invoke(protocols.clj:75)
	at clojure.core.protocols$fn__6684$G__6679__6697.invoke(protocols.clj:13)
	at clojure.core$reduce.invokeStatic(core.clj:6545)
	at clojure.core$into.invokeStatic(core.clj:6610)
	at clojure.walk$walk.invokeStatic(walk.clj:49)
	at clojure.walk$postwalk.invokeStatic(walk.clj:52)
	at clojure.walk$postwalk.invoke(walk.clj:52)
	at clojure.core$partial$fn__4759.invoke(core.clj:2515)
	at clojure.core$map$fn__4785.invoke(core.clj:2644)
	at clojure.lang.LazySeq.sval(LazySeq.java:40)
	at clojure.lang.LazySeq.seq(LazySeq.java:49)
	at clojure.lang.RT.seq(RT.java:521)
	at clojure.lang.LazilyPersistentVector.create(LazilyPersistentVector.java:44)
	at clojure.core$vec.invokeStatic(core.clj:377)
	at clojure.walk$walk.invokeStatic(walk.clj:45)
	at clojure.walk$postwalk.invokeStatic(walk.clj:52)
	at clojure.walk$postwalk.invoke(walk.clj:52)
	at clojure.core$partial$fn__4759.invoke(core.clj:2515)
	at clojure.core$map$fn__4785.invoke(core.clj:2646)
	at clojure.lang.LazySeq.sval(LazySeq.java:40)
	at clojure.lang.LazySeq.seq(LazySeq.java:49)
	at clojure.lang.Cons.next(Cons.java:39)
	at clojure.lang.RT.next(RT.java:688)
	at clojure.core$next__4341.invokeStatic(core.clj:64)
	at clojure.core.protocols$fn__6755.invokeStatic(protocols.clj:168)
	at clojure.core.protocols$fn__6755.invoke(protocols.clj:124)
	at clojure.core.protocols$fn__6710$G__6705__6719.invoke(protocols.clj:19)
	at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:31)
	at clojure.core.protocols$fn__6738.invokeStatic(protocols.clj:75)
	at clojure.core.protocols$fn__6738.invoke(protocols.clj:75)
	at clojure.core.protocols$fn__6684$G__6679__6697.invoke(protocols.clj:13)
	at clojure.core$reduce.invokeStatic(core.clj:6545)
	at clojure.core$into.invokeStatic(core.clj:6610)
	at clojure.walk$walk.invokeStatic(walk.clj:49)
	at clojure.walk$postwalk.invokeStatic(walk.clj:52)
	at clojure.walk$postwalk.invoke(walk.clj:52)
	at metabase.query_processor.middleware.resolve$resolve_tables.invokeStatic(resolve.clj:403)
	at metabase.query_processor.middleware.resolve$resolve_tables.invoke(resolve.clj:384)
	at metabase.query_processor.middleware.resolve$resolve.invokeStatic(resolve.clj:411)
	at metabase.query_processor.middleware.resolve$resolve.invoke(resolve.clj:408)
	at metabase.query_processor.middleware.resolve$resolve_middleware$fn__28067.invoke(resolve.clj:422)
	at clojure.core$comp$fn__4727.invoke(core.clj:2460)
	at metabase.query_processor.middleware.expand$expand_middleware$fn__29767.invoke(expand.clj:601)
	at clojure.core$comp$fn__4727.invoke(core.clj:2460)
	at clojure.core$comp$fn__4727.invoke(core.clj:2460)
	at clojure.core$comp$fn__4727.invoke(core.clj:2460)
	at metabase.models.card$mbql_permissions_path_set.invokeStatic(card.clj:174)
	at metabase.models.card$mbql_permissions_path_set.doInvoke(card.clj:137)
	at clojure.lang.RestFn.invoke(RestFn.java:464)
	at metabase.models.card$query_perms_set.invokeStatic(card.clj:200)
	at metabase.models.card$query_perms_set.doInvoke(card.clj:191)
	at clojure.lang.RestFn.invoke(RestFn.java:464)
	at metabase.models.card$maybe_update_read_permissions.invokeStatic(card.clj:315)
	at metabase.models.card$maybe_update_read_permissions.invoke(card.clj:302)
	at metabase.models.card$pre_update.invokeStatic(card.clj:351)
	at metabase.models.card$pre_update.invoke(card.clj:349)
	at clojure.core$comp$fn__4727.invoke(core.clj:2460)
	at toucan.models$fn__3985$G__3904__3990.invoke(models.clj:178)
	at toucan.models$do_pre_update.invokeStatic(models.clj:330)
	at toucan.models$do_pre_update.invoke(models.clj:325)
	at toucan.db$update_BANG_.invokeStatic(db.clj:479)
	at toucan.db$update_BANG_.invoke(db.clj:462)
	at metabase.db.migrations$repopulate_card_read_permissions$fn__32982.invoke(migrations.clj:396)
	at clojure.core$run_BANG_$fn__7276.invoke(core.clj:7393)
	at clojure.core$completing$fn__6948.invoke(core.clj:6585)
	at clojure.core$map$fn__4781$fn__4782.invoke(core.clj:2633)
	at clojure.java.jdbc$init_reduce_rs.invokeStatic(jdbc.clj:1120)
	at clojure.java.jdbc$init_reduce_rs.invoke(jdbc.clj:1112)
	at clojure.java.jdbc$reducible_result_set_STAR_$reify__1874.reduce(jdbc.clj:1142)
	at clojure.core$reduce.invokeStatic(core.clj:6544)
	at clojure.core$reduce.invoke(core.clj:6527)
	at clojure.java.jdbc$query_reducer$fn__1880$fn__1883.invoke(jdbc.clj:1177)
	at clojure.java.jdbc$execute_query_with_params.invokeStatic(jdbc.clj:994)
	at clojure.java.jdbc$execute_query_with_params.invoke(jdbc.clj:987)
	at clojure.java.jdbc$db_query_with_resultset_STAR_.invokeStatic(jdbc.clj:1016)
	at clojure.java.jdbc$db_query_with_resultset_STAR_.invoke(jdbc.clj:996)
	at clojure.java.jdbc$reducible_query$reify__1895.reduce(jdbc.clj:1272)
	at clojure.core$transduce.invokeStatic(core.clj:6600)
	at clojure.core.Eduction.reduce(core.clj:7360)
	at clojure.core$reduce.invokeStatic(core.clj:6544)
	at clojure.core$run_BANG_.invokeStatic(core.clj:7388)
	at clojure.core$run_BANG_.invoke(core.clj:7388)
	at metabase.db.migrations$repopulate_card_read_permissions.invokeStatic(migrations.clj:394)
	at metabase.db.migrations$repopulate_card_read_permissions.invoke(migrations.clj:393)
	at metabase.db.migrations$run_migration_if_needed_BANG_.invokeStatic(migrations.clj:50)
	at metabase.db.migrations$run_migration_if_needed_BANG_.invoke(migrations.clj:41)
	at metabase.db.migrations$run_all_BANG_.invokeStatic(migrations.clj:70)
	at metabase.db.migrations$run_all_BANG_.invoke(migrations.clj:64)
	at clojure.lang.Var.invoke(Var.java:375)
	at metabase.db$run_data_migrations_BANG_.invokeStatic(db.clj:390)
	at metabase.db$run_data_migrations_BANG_.invoke(db.clj:385)
	at metabase.db$setup_db_BANG_.invokeStatic(db.clj:401)
	at metabase.db$setup_db_BANG_.doInvoke(db.clj:392)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at metabase.core$init_BANG_.invokeStatic(core.clj:154)
	at metabase.core$init_BANG_.invoke(core.clj:133)
	at metabase.core$start_normally.invokeStatic(core.clj:258)
	at metabase.core$start_normally.invoke(core.clj:251)
	at metabase.core$_main.invokeStatic(core.clj:279)
	at metabase.core$_main.doInvoke(core.clj:274)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at metabase.core.main(Unknown Source)
04-09 09:17:51 ERROR metabase.core :: Metabase Initialization FAILED:  Query expansion failed: could not find table 26 (FK ID = 418). Resolved tables ([fk-id table-id]): ([nil 47])
04-09 09:17:51 INFO metabase.core :: Metabase Shutting Down ...
04-09 09:17:51 INFO metabase.core :: Metabase Shutdown COMPLETE

```

- Your operating system: Ubuntu 16.04
- Your databases: MySQL 5.6
- Metabase version: 0.28.5
- Metabase hosting environment: Ubuntu
- Metabase internal database: MySQL on RDS

Any solutions?

**Comment by zaiddabaeen** [comment]
I got this issue fixed:

The breaking change is between 0.28.4 to 0.28.5.

I gradually kept on testing if I can update to 0.28.5 from version between 0.27.2 until 0.28.4. Had to go through  0.28.0 0.28.2 0.28.4 migrations until I could do the 0.28.5 migration.

**Comment by adrian17** [comment]
I don't think this should be closed. We were planning to upgrade from 0.28.3 to 0.28.5 and it'd be nice (for us and everyone else) if we didn't need to manually go through 0.28.4.
Also, some people may not be lucky enough to quickly find this walkaround.

**Comment by und3fined** [comment]
My catch:

```
[metabase@the-bi script]$ java -Xmx1536m -Dlog4j.configuration=file:/home/metabase/script/log4j.properties -jar metabase.jar
04-10 03:53:39 INFO metabase.util :: Loading Metabase...
04-10 03:53:50 INFO util.encryption :: DB details encryption is ENABLED for this Metabase instance. 🔐
See http://www.metabase.com/docs/latest/operations-guide/start.html#encrypting-your-database-connection-details-at-rest for more information.
04-10 03:54:07 INFO metabase.core :: Starting Metabase in STANDALONE mode
04-10 03:54:07 INFO metabase.core :: Launching Embedded Jetty Webserver with config:
 {:port 3000}

04-10 03:54:08 INFO metabase.core :: Starting Metabase version v0.28.5 (fa5cb8a release-0.28.5) ...
04-10 03:54:08 INFO metabase.core :: System timezone is 'UTC' ...
04-10 03:54:10 INFO metabase.core :: Setting up and migrating Metabase DB. Please sit tight, this may take a minute...
04-10 03:54:10 INFO metabase.db :: Verifying postgres Database Connection ...
04-10 03:54:11 INFO metabase.db :: Verify Database Connection ...  ✅
04-10 03:54:11 INFO metabase.db :: Running Database Migrations...
04-10 03:54:11 INFO metabase.db :: Setting up Liquibase...
04-10 03:54:11 INFO metabase.db :: Liquibase is ready.
04-10 03:54:11 INFO metabase.db :: Checking if Database has unrun migrations...
04-10 03:54:15 INFO metabase.db :: Database Migrations Current ...  ✅
com.mchange.v2.cfg.DelayedLogItem [ level -> FINE, text -> "The configuration file for resource identifier 'hocon:/reference,/application,/c3p0,/' could not be found. Skipping.", exception -> null]
04-10 03:54:15 INFO db.migrations :: Running all necessary data migrations, this may take a minute.
04-10 03:54:15 INFO db.migrations :: Running data migration 'repopulate-card-read-permissions'...
04-10 03:54:16 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
java.lang.Exception: Failed to resolve fields: too many iterations.
	at metabase.query_processor.middleware.resolve$resolve_fields.invokeStatic(resolve.clj:329)
	at metabase.query_processor.middleware.resolve$resolve_fields.invoke(resolve.clj:325)
	at metabase.query_processor.middleware.resolve$resolve.invokeStatic(resolve.clj:411)
	at metabase.query_processor.middleware.resolve$resolve.invoke(resolve.clj:408)
	at metabase.query_processor.middleware.resolve$resolve_middleware$fn__28067.invoke(resolve.clj:422)
	at clojure.core$comp$fn__4727.invoke(core.clj:2460)
	at metabase.query_processor.middleware.expand$expand_middleware$fn__29767.invoke(expand.clj:601)
	at clojure.core$comp$fn__4727.invoke(core.clj:2460)
	at clojure.core$comp$fn__4727.invoke(core.clj:2460)
	at clojure.core$comp$fn__4727.invoke(core.clj:2460)
	at metabase.models.card$mbql_permissions_path_set.invokeStatic(card.clj:174)
	at metabase.models.card$mbql_permissions_path_set.doInvoke(card.clj:137)
	at clojure.lang.RestFn.invoke(RestFn.java:464)
	at metabase.models.card$query_perms_set.invokeStatic(card.clj:200)
	at metabase.models.card$query_perms_set.doInvoke(card.clj:191)
	at clojure.lang.RestFn.invoke(RestFn.java:464)
	at metabase.models.card$maybe_update_read_permissions.invokeStatic(card.clj:315)
	at metabase.models.card$maybe_update_read_permissions.invoke(card.clj:302)
	at metabase.models.card$pre_update.invokeStatic(card.clj:351)
	at metabase.models.card$pre_update.invoke(card.clj:349)
	at clojure.core$comp$fn__4727.invoke(core.clj:2460)
	at toucan.models$fn__3985$G__3904__3990.invoke(models.clj:178)
	at toucan.models$do_pre_update.invokeStatic(models.clj:330)
	at toucan.models$do_pre_update.invoke(models.clj:325)
	at toucan.db$update_BANG_.invokeStatic(db.clj:479)
	at toucan.db$update_BANG_.invoke(db.clj:462)
	at metabase.db.migrations$repopulate_card_read_permissions$fn__32982.invoke(migrations.clj:396)
	at clojure.core$run_BANG_$fn__7276.invoke(core.clj:7393)
	at clojure.core$completing$fn__6948.invoke(core.clj:6585)
	at clojure.core$map$fn__4781$fn__4782.invoke(core.clj:2633)
	at clojure.java.jdbc$init_reduce_rs.invokeStatic(jdbc.clj:1120)
	at clojure.java.jdbc$init_reduce_rs.invoke(jdbc.clj:1112)
	at clojure.java.jdbc$reducible_result_set_STAR_$reify__1874.reduce(jdbc.clj:1142)
	at clojure.core$reduce.invokeStatic(core.clj:6544)
	at clojure.core$reduce.invoke(core.clj:6527)
	at clojure.java.jdbc$query_reducer$fn__1880$fn__1883.invoke(jdbc.clj:1177)
	at clojure.java.jdbc$execute_query_with_params.invokeStatic(jdbc.clj:994)
	at clojure.java.jdbc$execute_query_with_params.invoke(jdbc.clj:987)
	at clojure.java.jdbc$db_query_with_resultset_STAR_.invokeStatic(jdbc.clj:1016)
	at clojure.java.jdbc$db_query_with_resultset_STAR_.invoke(jdbc.clj:996)
	at clojure.java.jdbc$reducible_query$reify__1895.reduce(jdbc.clj:1272)
	at clojure.core$transduce.invokeStatic(core.clj:6600)
	at clojure.core.Eduction.reduce(core.clj:7360)
	at clojure.core$reduce.invokeStatic(core.clj:6544)
	at clojure.core$run_BANG_.invokeStatic(core.clj:7388)
	at clojure.core$run_BANG_.invoke(core.clj:7388)
	at metabase.db.migrations$repopulate_card_read_permissions.invokeStatic(migrations.clj:394)
	at metabase.db.migrations$repopulate_card_read_permissions.invoke(migrations.clj:393)
	at metabase.db.migrations$run_migration_if_needed_BANG_.invokeStatic(migrations.clj:50)
	at metabase.db.migrations$run_migration_if_needed_BANG_.invoke(migrations.clj:41)
	at metabase.db.migrations$run_all_BANG_.invokeStatic(migrations.clj:70)
	at metabase.db.migrations$run_all_BANG_.invoke(migrations.clj:64)
	at clojure.lang.Var.invoke(Var.java:375)
	at metabase.db$run_data_migrations_BANG_.invokeStatic(db.clj:390)
	at metabase.db$run_data_migrations_BANG_.invoke(db.clj:385)
	at metabase.db$setup_db_BANG_.invokeStatic(db.clj:401)
	at metabase.db$setup_db_BANG_.doInvoke(db.clj:392)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at metabase.core$init_BANG_.invokeStatic(core.clj:154)
	at metabase.core$init_BANG_.invoke(core.clj:133)
	at metabase.core$start_normally.invokeStatic(core.clj:258)
	at metabase.core$start_normally.invoke(core.clj:251)
	at metabase.core$_main.invokeStatic(core.clj:279)
	at metabase.core$_main.doInvoke(core.clj:274)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at metabase.core.main(Unknown Source)
04-10 03:54:16 ERROR metabase.core :: Metabase Initialization FAILED:  Failed to resolve fields: too many iterations.
04-10 03:54:16 INFO metabase.core :: Metabase Shutting Down ...
04-10 03:54:16 INFO metabase.core :: Metabase Shutdown COMPLETE
```

**Comment by camsaul** [comment]
@und3fined can you try my PR #7313 and confirm whether it fixes your issue?

**Comment by camsaul** [comment]
Fixed by #7313

**Comment by und3fined** [comment]
@camsual: I will try in new version (0.29.0)
